single-line-diagram-core/src/test/java/com/powsybl/sld/raw/TestCaseComplexCoupling.java

```diff
@@ -50,7 +51,11 @@ public void setUp() {
     @Test
     public void test() {
         VoltageLevelGraph g = rawGraphBuilder.buildVoltageLevelGraph("vl", true);
+
+        layoutParameters.setAdaptCellHeightToContent(true);
         voltageLevelGraphLayout(g);
+
         assertEquals(toString("/TestCaseComplexCoupling.json"), toJson(g, "/TestCaseComplexCoupling.json"));
+        assertEquals(toString("/TestCaseComplexCoupling.svg"), toSVG(g, "/TestCaseComplexCoupling.svg", getRawLabelProvider(g), new BasicStyleProvider()));
```

Comment:
You can keep only one of them I'd say; depending on what you want to check you can keep only the json or only the svg.